components/datepicker/modules/month-year-picker.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="d-datepicker__month-year-picker">
+    <div>
+      <button
+        type="button"
+        :aria-label="prevYearLabel"
+        @click="handleYear(false)"
+      >
+        <dt-icon
+          name="chevrons-left"
+          size="300"
+          :description="prevYearLabel"
```

Comment:
This isn't a `dt-icon` prop, so this wouldn't do anything, you can remove it.

Reply:
`test a11y fix` leftover